.github/CODEOWNERS

```diff
@@ -5,3 +5,6 @@
 /client/libra-dev/ @thefallentree @xli
 # Subscribe to LCS format changes
 /testsuite/generate-format/tests/staged/libra.yaml @thefallentree @matbd @xli
+# Subscribe to TCB dependency summary changes
+/summaries/summary-lsr.toml @davidiw
+/summaries/summary-lec.toml @davidiw
```

Comment:
Do we want an owner for the release summary?

Reply:
I was only going to enforce the TCB with the extra review as a first step. We can enforce the release binaries later if this goes well.